source/api-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _rm-rest-api:
+
+============================
+Relational Migrator REST API
+============================
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. meta:: 
+   :description: Learn about the MongoDB Relational Migrator REST API and how to use it.
+   :keywords: data migration, rest api, open api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The Relational Migrator REST API provides you with programmatic access 
+to Relation Migrator with HTTPS endpoints. The REST API is a middleware 
+service that sits between your application and your source and destination 
+databases that you can send requests to facilitate and manage your sync 
+jobs.
+
+You don't need to install any additional extensions or packages to use 
+the Relation Migrator REST API. The functionality is built into the core 
```

Comment:
Can you avoid "core"?
And, "built into"? Sounds a bit too "marketing informal". I'd go with something like "The Relational Migrator REST API is part of the Relational Migrator product..." or something like that.